### DISCUSSION TITLE
Elsa 3.1.3 Designer development checks custom activities for roles and permissions to interact with the user management system?

### DISCUSSION THREAD

**Discussion by shxylgithub** [elsa-workflows/elsa-core#5737] [General]
Elsa 3.1.3 Designer development checks custom activities for roles and permissions to interact with the user management system?

**Comment by glime-ai** [comment]
Based on the information provided, there is no direct mention of Elsa 3.1.3 Designer performing checks for roles and permissions specifically for custom activities in the context of interacting with a user management system. However, there is a GitHub issue asking about setting roles and user approval permissions for nodes in Elsa 3.1.3 Designer, which suggests that there is some level of role and permission management within Elsa workflows.

To get a definitive answer on whether Elsa 3.1.3 Designer supports roles and permissions checks for custom activities, you would typically need to refer to the official documentation or reach out to the community or support channels for clarification. Since the documentation is a work in progress and the provided context does not include a direct answer, it is recommended to check the latest documentation or inquire with the Elsa community or support for the most current information.

If you need further assistance or have specific requirements for roles and permissions with custom activities in Elsa, please provide additional details so that I can assist you more effectively.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
